src/main/kotlin/net/atos/zac/app/audit/converter/documenten/AuditBesluitInformatieobjectConverter.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.audit.converter.documenten
+
+import jakarta.inject.Inject
+import net.atos.client.or.shared.util.URIUtil
+import net.atos.client.zgw.brc.BrcClientService
+import net.atos.client.zgw.brc.model.generated.BesluitInformatieObject
+import net.atos.client.zgw.shared.model.audit.besluiten.BesluitInformatieobjectWijziging
+import net.atos.zac.app.audit.model.RESTHistorieRegel
+
+class AuditBesluitInformatieobjectConverter @Inject constructor(
+    private val brcClientService: BrcClientService
+) {
+
+    fun convert(wijziging: BesluitInformatieobjectWijziging): List<RESTHistorieRegel> {
+        val oud = wijziging.oud
```

Comment:
Not really a need for these vals or is there?

Reply:
Wanted to have the same logic as in `AuditEnkelvoudigInformatieobjectConverter` in case we implement more complex conversion.

But at the moment we can embed these in the listOf below. Will do just that

Reply:
Applied your suggestion